tests/test_as_path_verification.c

```diff
@@ -14,108 +14,92 @@
 #include <string.h>
 #include <assert.h>
 
-static struct aspa_record *create_aspa_record(uint32_t cas, uint32_t *provider_asns, size_t provider_count)
-{
-	struct aspa_record *aspa_record = lrtr_malloc(sizeof(struct aspa_record));
-	aspa_record->customer_asn = cas;
-
-	size_t size = provider_count * sizeof(uint32_t);
-	aspa_record->provider_asns = lrtr_malloc(size);
-	memcpy(aspa_record->provider_asns, provider_asns, size);
 
-	aspa_record->provider_count = provider_count;
+#define ASNS(...) (uint32_t []) { __VA_ARGS__ }
 
-	return aspa_record;
+#define RECORD(cas, providers) (struct aspa_record) { \
```

Comment:
Maybe we should create a test header for ASPA that defines `RECORD`, and `ASNS`...

Reply:
For now, I'd like to keep it like it is